.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+
+node_js:
+  - "stable"
+
+cache:
+  directories:
+  - node_modules
+
+script:
+  - npm run test:coverage
```

Comment:
@ezrogha  could put a space at the end of this file

Reply:
Thank you @felixkiryowa 
I have worked on it